pyblish_qml/__init__.py

```diff
@@ -5,12 +5,12 @@
 )
 
 
-def show(parent=None, targets=None, modal=None, foster=None):
+def show(parent=None, targets=None, modal=None):
```

Comment:
Even though `foster=` doesn't do anything, we should keep it around for a few versions, just to ensure it doesn't break people's existing installations.

Reply:
I'll put a warning if `foster is not None`
Thanks !